pkgs/by-name/fl/fleeting-plugin-aws/package.nix

```diff
@@ -0,0 +1,56 @@
+{
+  lib,
+  buildGoModule,
+  fetchFromGitLab,
+  nix-update-script,
+  versionCheckHook,
+}:
+
+let
+  # See https://gitlab.com/gitlab-org/fleeting/plugins/aws/-/blob/v1.0.0/Makefile?ref_type=tags#L14.
+  #
+  # Couldn't find a way to substitute "go list ." into "ldflags".
+  ldflagsPackageVariablePrefix = "gitlab.com/gitlab-org/fleeting/plugins/aws";
+in
+buildGoModule rec {
+  pname = "fleeting-plugin-aws";
+  version = "1.0.0";
+
+  src = fetchFromGitLab {
+    owner = "gitlab-org/fleeting/plugins";
+    repo = "aws";
+    rev = "refs/tags/v${version}";
+    hash = "sha256-8vEduf+xh9R3+GoouXJS2h/ELlzKXDmLBLekaXGn7SE=";
+  };
+
+  vendorHash = "sha256-bfEzPPP280peOK4Jyu1fyfFCaFnRLoPmsjJ+G1BoVW4=";
+
+  subPackages = [ "cmd/fleeting-plugin-aws" ];
+
+  # See https://gitlab.com/gitlab-org/fleeting/plugins/aws/-/blob/v1.0.0/Makefile?ref_type=tags#L20-22.
+  #
+  # Needed for "fleeting-plugin-aws version" to not show "dev".
+  ldflags = [
+    "-X ${ldflagsPackageVariablePrefix}.NAME=${pname}"
+    "-X ${ldflagsPackageVariablePrefix}.VERSION=v${version}"
+    "-X ${ldflagsPackageVariablePrefix}.REVISION=${src.rev}"
+  ];
+
+  nativeInstallCheckInputs = [ versionCheckHook ];
+
+  versionCheckProgramArg = "version";
+
+  passthru = {
+    updateScript = nix-update-script { };
+  };
+
+  meta = {
+    description = "This is a fleeting plugin for AWS.";
+    homepage = "https://gitlab.com/gitlab-org/fleeting/plugins/aws";
+    license = lib.licenses.mit;
+    mainProgram = "fleeting-plugin-aws";
+    platforms = lib.platforms.all;
+    # TODO: Find maintainer(s).
+    maintainers = with lib.maintainers; [ ];
```

Comment:
Honestly you'd be surprised sometimes how little work maintenance can be, especially with an update script in place. It'd be better if you just put yourself here for now, and you can always ask for package adoption later

Reply:
I assure you that @r-ryantm is not *that* diligent :p

Reply:
To be honest, I don't use the majority of the packages I maintain. But for me, it's not something that really takes up much of my time, which is why I say you should try maintaining it first and see what the actual maintenance burden is — if it gets too much, you can always ask for maintainers *when that time comes* instead of worrying about it now